index.js

```diff
@@ -14,6 +14,10 @@ function Attempt(instance, end) {
   var command = [];
   command.push('/usr/bin/sudo');
   command.push('-n');
+
+  // Preserve user environment 
```

Comment:
Can you add a colon to the end of the comment and remove the leading and trailing newlines?